x-pack/plugins/canvas/canvas_plugin_src/functions/common/alterColumn.ts

```diff
@@ -43,9 +50,9 @@ export function alterColumn() {
       }
 
       const name = args.name || column.name;
-      const type = args.type || column.type;
+      const type = (args.type || column.type) as DatatableColumnType;
```

Comment:
`DatatableColumnType` admits `'date'` but the `Arguments` in `alterColumn` does not, not sure if it's an intended mismatch. On the other hand, the latter admits both `null` and `'null'`, maybe that's also an intentional difference, not clear to me